.travis.yml

```diff
@@ -15,8 +15,8 @@ before_install:
 
 install:
   - export GIT_FULL_HASH=`git rev-parse HEAD`
-  - conda create --yes -n testenv --file requirements/run.txt --quiet
-  - source activate testenv
+  - conda install --file requirements/run
+  - conda install pytest
```

Comment:
maybe it would be a good idea to have a `requirements/test` file. Muliple `--file` arguments can be passed in.